.github/workflows/test.yml

```
@@ -360,6 +360,35 @@ jobs:
       - name: backend | Lint
         run: docker run --rm gradido/backend:test yarn run lint
 
+  ##############################################################################
+  # JOB: LOCALES BACKEND #######################################################
+  ##############################################################################
+  locales_frontend:
```

Should the name not be locales_backend instead of locales_frontend?
```suggestion
  locales_backend:
```

---

Thank you.
This is fixed in https://github.com/gradido/gradido/pull/2693/commits/ee5aea4c85dc85b6b42ac794c2edf1b52708379d.